### PR TITLE
LEARNER-7441 Test - fix security vulnerability alert by github in edx-app-test repo

### PR DESCRIPTION
**Pull request by nedo**
Thanks @staubina  reporting `lodash' package security vulnerability in `tests/package-lock.json` file.

Need to update `lodash` package

### CODE REVIEW

**Comment by nedo** [comment]
@farhan-arshad-dev  can you please review this PR ? 

cc @mulby

File: tests/android/pages/android_register.py [hunk 1]
```diff
@@ -538,19 +538,21 @@ def get_selected_textview_in_spinner(self, spinner):
...
-                return texts
+                output = texts
             else:
                 self.log.info('0 text views found in spinner')
 
         else:
-            return None
+            output = None
+
+        return output
```
Comment (farhan-arshad-dev) [inline tests/android/pages/android_register.py]:
``` python
        texts = spinner.find_elements_by_class_name(android_elements.all_textviews)
        if texts is not None:

            country = len(texts)
            if country > 0:
                self.log.info('Total {} text views found in spinner'.format(len(texts)))
                return texts
            else:
                self.log.info('0 text views found in spinner')

        return None
````

IMO, This Code is more optimised.

File: tests/common/globals.py [hunk 2]
```diff
@@ -168,11 +169,13 @@ def get_all_views_on_screen(self, driver, target_elements):
                     for view in all_views:
                         self.project_log.info('{}. {}, with value - {}'.format(self.index, view, view.text))
                         self.index += 1
-                    return all_views
+                    output = all_views
                 else:
                     self.project_log.info('0 {} found on screen'.format(target_elements))
             else:
-                return None
+                output = None
```
Comment (farhan-arshad-dev) [inline tests/common/globals.py]: Remove the `else` part and just return None. After that, there is no need of `output` variable.

File: tests/common/globals.py [hunk 3]
```diff
@@ -168,11 +169,13 @@ def get_all_views_on_screen(self, driver, target_elements):
                     for view in all_views:
                         self.project_log.info('{}. {}, with value - {}'.format(self.index, view, view.text))
                         self.index += 1
-                    return all_views
+                    output = all_views
```
Comment (farhan-arshad-dev) [inline tests/common/globals.py]: After removing the `output` we can user return statement.

File: tests/common/globals.py [hunk 4]
```diff
@@ -214,11 +218,13 @@ def get_all_views_on_ios_screen(self, driver, target_elements):
                         self.project_log.info('{}. {}, with value - {}'.format(self.index, view, view.text))
                         self.index += 1
 
-                    return all_views
+                    output = all_views
                 else:
                     self.project_log.info('0 {} found on screen'.format(target_elements))
             else:
-                return None
+                output = None
```
Comment (farhan-arshad-dev) [inline tests/common/globals.py]: same.

**Comment by farhan-arshad-dev** [comment]
@nedo Completed first pass for Code review.

File: tests/android/pages/android_register.py (hunk 1 above)
Comment (nedo) [reply tests/android/pages/android_register.py]: done

File: tests/common/globals.py (hunk 3 above)
Comment (nedo) [reply tests/common/globals.py]: done

File: tests/common/globals.py (hunk 4 above)
Comment (nedo) [reply tests/common/globals.py]: done

**Pull request merged**